src/components/Register.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Input,
+} from 'reactstrap';
+import axios from 'axios';
+
+class Register extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  submitForm = (e) => {
+    e.preventDefault();
+    const { username, password } = this.state;
+    const url = 'http://localhost:8000/users/register';
+
+    axios.post(url, {
+      username: username,
+      password: password
```

Comment:
when you key has the same name as the value, you can just write `{username, password}`